src/main/resources/db/migration/V7__Add_utbet_tom.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE UTBETALING_SPLEIS ADD COLUMN UTBET_TOM DATE;
```

Comment:
Er det en grunn til at vi ikke vil skrive den helt ut og kalle den `UTBETALT_TOM`?

Reply:
Enig med Eirik at vi bare burde gå for `UTBETALT_TOM` 😊